src/shared/src/types/bitcoin.rs

```diff
@@ -10,6 +30,7 @@ pub struct SelectedUtxosFeeRequest {
 }
 
 #[derive(CandidType, Deserialize, Clone, Eq, PartialEq, Debug)]
+#[serde(remote = "Self")]
```

Comment:
This allows us to have a custom serializer that calls the standard serilaizer, then the validation code.